templates/zerver/emails/confirm_new_email.source.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
 <p>{% trans %}Hi {{ user_name }},{% endtrans %}</p>
 
-<p>{% trans realm_uri=macros.link_tag(realm_uri) %}We received a request to change the email address for the Zulip account on {{ realm_uri }} from {{ old_email }} to {{ new_email }}. To confirm this change, please click below:{% endtrans %}
+<p>{% trans realm_uri=macros.link_tag(realm_uri), old_email=macros.email_tag(old_email), new_email=macros.email_tag(new_email) %}We received a request to change the email address for the Zulip account on {{ realm_uri }} from {{ old_email }} to {{ new_email }}. To confirm this change, please click below:{% endtrans %}
```

Comment:
Do we actually want `mailto`  links for these emails?  I don't think similar account security emails for other products do so, but we should check.

Reply:
![Screenshot from 2020-10-29 19-48-05](https://user-images.githubusercontent.com/7190633/97585967-b5b86100-1a1f-11eb-96aa-8bd6d9c3c98c.png)

Slack seems to be linking the email

Reply:
I tried Slack, Twitter, and Notion. None of them seems to be mentioning the email at all.

![Screenshot from 2020-10-29 20-02-45](https://user-images.githubusercontent.com/7190633/97589788-f6b27480-1a23-11eb-9115-582c1d70a212.png)

![Screenshot from 2020-10-29 20-02-27](https://user-images.githubusercontent.com/7190633/97589637-c8349980-1a23-11eb-8d49-56c4600231e0.png)

Reply:
Yeah, I think it's helpful to mention the email in case folks are doing forwarding, but we certainly don't have a reason to linkify it.